packages/lib-classifier/src/plugins/tasks/survey/components/components/Chooser/components/Choices/components/ChoiceButton.js

```diff
@@ -180,6 +187,7 @@ ChoiceButton.propTypes = {
   columnsCount: PropTypes.number,
   disabled: PropTypes.bool,
   hasFocus: PropTypes.bool,
+  index: PropTypes.number,
```

Comment:
In the interest of tree shaking, a suggestion to `import { bool, .... } from 'prop-types'` rather than import the entirety of PropTypes in this file.